cmd/doctor.go

```diff
@@ -552,14 +552,19 @@ func runDoctorCheckDBConsistency(ctx *cli.Context) ([]string, error) {
 		}
 	}
 
+	var (
+		rPullRequest = models.RealTableName("pull_request")
+		rIssue       = models.RealTableName("issue")
```

Comment:
This is not correct. Because the variables have been filled but all the settings haven't been filled.

I think we could have these variables, but there is a function to init all the variables and manually invoked after settings have been filled.

The name `tbPullRequest` maybe a better name rather than `rPullRequest`.